SampleDestination/hosting.json

```diff
@@ -0,0 +1,3 @@
+{
+  "server.urls": "http://destination.sample.com:80"
```

Comment:
The host name doesn't matter here, use *.
Also, 80 is likely to cause a conflict. try `http://*:5000/`